src/styles/style-helpers.spec.tsx

```diff
@@ -13,14 +13,81 @@
  * limitations under the License.
  */
 
-import { light } from '@sumup/design-tokens';
-
-import * as StyleHelpers from './style-helpers';
+/** @jsx jsx */
+import { jsx, css } from '@emotion/core';
+import { light, Theme } from '@sumup/design-tokens';
+
+import { create } from '../util/test-utils';
+
+import {
+  cx,
+  shadowSingle,
+  shadowDouble,
+  shadowTriple,
+  headingKilo,
+  headingMega,
+  headingGiga,
+  headingTera,
+  headingExa,
+  headingPeta,
+  headingZetta,
+  subHeadingKilo,
+  subHeadingMega,
+  textKilo,
+  textMega,
+  textGiga,
+  disableVisually,
+  hideVisually,
+  focusOutline,
+  clearfix,
+  hideScrollbar,
+  inputOutline,
+} from './style-helpers';
 
 describe('Style helpers', () => {
+  const red = (theme: Theme) => css`
+    color: ${theme.colors.r500};
+  `;
+  const green = (theme: Theme) => css`
+    color: ${theme.colors.g500};
+  `;
+
+  describe('cx', () => {
+    it('should call each style function with the theme', () => {
+      const actual = create(<div css={cx(red, green)} />);
+
+      expect(actual).toMatchInlineSnapshot(`
+        .circuit-0 {
+          color: #D23F47;
+          color: #8CC13F;
+        }
+
+        <div
+          class="circuit-0"
+        />
+      `);
+    });
+
+    it('should skip falsy style functions', () => {
+      const isGreen = false;
+
+      const actual = create(<div css={cx(red, isGreen && green)} />);
```

Comment:
Ah, now I understand the type that accepts `false` for example. But wouldn't it help to remove it from the type so that the TS compiler gives a warning?

And what if `isGreen && green` is `true`?

Reply:
Why/what should the TS compiler warn about?

The point is to allow for falsy values so that styles can be conditionally applied with ease. If `isGreen` is `true`, then the `green` style function is called and its styles are added to the element.

Reply:
Ah, I didn't understand that the pattern of passing expressions was an intended use case. I thought we wanted to prevent it.

But I think this makes sense! Conditional styles can be passed and they'll either be filtered out if falsy, or applied.

Someone could still potentially pass a truthy value that is not a function, but like you mentioned in https://github.com/sumup-oss/circuit-ui/pull/835#discussion_r590387101, we don't necessarily have to handle this (at least not in production).